src/main/java/service/impl/ReportWriterService.java

```diff
@@ -0,0 +1,26 @@
+package service.impl;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import service.FileWriterService;
+
+public class ReportWriterService implements FileWriterService {
+    private static final String HEADER_LINE = "fruit,quantity";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
+    @Override
+    public void writeToFile(List<String> fruitsBalanceReport) {
+        File report = new ReportFileCreatorImpl().createFile();

```

Comment:
it is redundant to create file first, cause if file does not exist and you try to write something into it, file will be created

Reply:
Done